src/internal.c

```diff
@@ -32386,6 +32390,10 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
             }
             else {
                 WOLFSSL_MSG("Could not verify suite validity, continue");
+                if (ret == MEMORY_E) {
```

Comment:
It looks like `VerifyServerSuite` above used to return `1` or `0` and new there are places it can also return negative error codes. Please adjust to not just handle `MEMORY_E`. Thanks

Reply:
This check should be moved up to line 32376 and should be if `ret < 0`. Please move the async block into it also.

Reply:
@dgarske, good point. Moved and changed to `ret < 0`